api/controllers/v1/bibliographic-metadata/get-record-format.js

```diff
@@ -0,0 +1,49 @@
+const BibliographicMetadataService = require('../../../services/BibliographicMetadataService');
+const ControllerService = require('../../../services/ControllerService');
+const MarcConvertorService = require('../../../services/MarcConvertorService');
+
+/**
+ * Get Bibliographic metadata by ID and format
+ * GET /bibliographic-metadata/:id/format/:format
+ */
+module.exports = async (req, res) => {
+  const { id, format } = req.params;
+
+  if (!id || !format) {
+    return res.badRequest({
+      message: 'ID and format parameters are required',
+    });
+  }
+
+  const bibliographicMetadata =
+    await BibliographicMetadataService.getMetadata(id);
```

Comment:
Comme on ne l'utilise qu'une seule fois et au vu de sa simplicité, il vaut mieux ne pas avoir de service ici :
```suggestion
(await TBibliographicMetadata.findOne({id , metadataStatus: 'registered'})) ?? null
```

Reply:
Il est fort possible que pour les routes du serveur OAI cela soit utile ce service soit réutilisé. Est ce que je le change quand même ?